CONTRIBUTING.md

```diff
@@ -10,6 +10,9 @@ You can create as many accounts as needed in order to test your changes directly
 ## Code of Conduct
 This project and everyone participating in it is governed by the Expensify [Code of Conduct](https://github.com/Expensify/Expensify.cash/blob/main/CODE_OF_CONDUCT.md). By participating, you are expected to uphold this code. Please report unacceptable behavior to [contributors@expensify.com](mailto:contributors@expensify.com).
 
+## Restrictions
+We are not currently hiring contractors in US sanctioned countries, including Russia, at this time.
```

Comment:
Could we add a link here or something to the official list?

Reply:
Asked in [a slack thread](https://expensify.slack.com/archives/C06FX871V/p1621471359017600), updated this to specify the countries.